### ISSUE TITLE
Contract Import always fails with "Failed parsing config     1: EE108: The 'name' field in your config file must have the following pattern..."

### ISSUE THREAD

**Issue by moose-code** [enviodev/hyperindex#182]
**Describe the bug**
Contract import seems to always fail with this error:
```
Error: Failed cli execution

Caused by:
    0: Failed parsing config
    1: EE108: The 'name' field in your config file must have the following pattern: It must start with a letter or underscore. It can contain letters, numbers, and underscores (no spaces). It must have a maximum length of 63 characters
```

**To Reproduce**

Simply use envio init on v2.2.2 and import a contract on Base network (suspected issue for all networks)

**Expected behavior**
It shouldn't fail, it should autogenerate the indexer. 

**Screenshots**
=![Screenshot 2024-09-06 at 14 41 23](https://github.com/user-attachments/assets/4ca4e66e-1416-40d9-b3e7-7b9b2edee83e)

**Local (please complete the following information):**
 - Envio v2.2.2

**Additional context**
Suspected that maybe having spaces in the name makes it fail

**Comment by moose-code** [comment]
For example I used `0x4200000000000000000000000000000000000016` on Base network

**Comment by DZakh** [comment]
Solved by the https://github.com/enviodev/hyperindex/pull/184

**Comment by JonoPrest** [comment]
Closed by #184
